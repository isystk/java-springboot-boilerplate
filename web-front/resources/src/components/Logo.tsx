import React, { VFC } from 'react'
import { Link } from 'react-router-dom'
import { Url } from '@/constants/url'

const Logo: VFC = () => {
  const DEAULT_TITLE = 'amplify-reactjs-sample'

  return (
    <div className="App-logo">
      <Link to={Url.Top}>{DEAULT_TITLE}</Link>
    </div>
  )
}

export default Logo
