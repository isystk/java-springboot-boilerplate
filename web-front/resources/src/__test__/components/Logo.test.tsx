import React from 'react'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Logo from '@/components/Logo'

test('Logo', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Logo />
    </BrowserRouter>
  )
  expect(component).toMatchSnapshot()
})
