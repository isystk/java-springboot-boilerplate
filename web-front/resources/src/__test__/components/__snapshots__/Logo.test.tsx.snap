// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<div
  className="App-logo"
>
  <a
    href="/"
    onClick={[Function]}
  >
    amplify-reactjs-sample
  </a>
</div>
`;
